libraries/AP_Parachute/AP_Parachute.cpp

```diff
@@ -59,6 +59,23 @@ const AP_Param::GroupInfo AP_Parachute::var_info[] = {
     // @Increment: 1
     // @User: Standard
     AP_GROUPINFO("DELAY_MS", 5, AP_Parachute, _delay_ms, AP_PARACHUTE_RELEASE_DELAY_MS),
+
+    // @Param: EMERGENCY
+    // @DisplayName: Emergency parachute enable
+    // @Description: Releases the parachute when an emergency occurs
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Standard
+    AP_GROUPINFO("EMERGENCY", 6, AP_Parachute, _emergency_enabled, AP_PARACHUTE_EMERGENCY_DEFAULT),
```

Comment:
the name CHUTE_EMERGENCY name isn't good, as it doesn't have an obvious link to CHUTE_CRT_SINK. 
I think we can just remove it, and instead make CHUTE_CRT_SINK=0 mean disable. Then default it to zero.

Reply:
Agreed